scripts/predictor/README.md

```diff
@@ -0,0 +1,59 @@
+# WHAT IS THIS SCRIPT SET FOR?
+
+This set of scripts is intended to help devs to find out if the contracts of a given commit / branch in a repo will reproduce exactly the contracts deployed in the blockchain.
+These scripts are categorized into four .js files for convenient separate execution.
+This can be used as a tool to predict if the contract code in certain repo / branch / commit will verify in a block explorer.
+
+## Scripts and JSON Files
+
+The folder ./scripts/predictor have four .js files and may have several .json files, but one them is invariant: contract_config.json
+
+These scripts will work only provided that there exist a folder in the repo named ./scripts/contractInteraction contentive of the .json files listing all the Sovryn's deployed contracts.
```

Comment:
```suggestion
These scripts will work only provided that there exists a folder in the repo named ./scripts/contractInteraction contentive of the .json files listing all the Sovryn's deployed contracts.
```
pls consider putting all settings - paths etc. in e.g. .deployment-verifier.json or .ts/.js